src/main/webapp/index.html

```diff
@@ -1,19 +1,41 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="utf-8">
+        <script src="https://kit.fontawesome.com/848c0f5fe3.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" type="text/css" href="../style.css">
+        <script src="script.js"></script>
+        <title>Logo: Shopping List</title>
+    </head>
+    <body style="background-color: #D0E4CF">
+        <div>
+            <h1 style="color: #0A6644">Welcome Back, John Doe</h1>
+            <h2 style="color: #0A6644">What's On Your Mind?</h2>
+        </div>
+        <br>
+        <br>
+        <div>
+            <form>
+                <h2 style="color: #0A6644">Add to your Shopping List</h2>
+                <input id="item-name" type="text" name="text-input"  placeholder="  Enter Item.." style="border-radius: 20px; color: #0A6644; width: 300px; height: 30px;">
+                <input type="button" value = "submit" onclick="addToCart()"/>                
+                
+            </form>
+            <p>Calculate item price and total prices</p>
+            <button onclick="calculate();">Calculate</button>
+        </div>
+        <br>
+        <br>
+        <div>
+            <h2 style="color: #0A6644">Your Shopping List</h2>
+            <li style="color: #0A6644">Item 1</li>
+            <li style="color: #0A6644">Item 2</li>
+        </div>
+
+        <br>
+        <br>
+        <!-- simply a link to the comparison page for now 
+            (should display real prices after entering shopping list) -->
+        <a href="compare.html">See Comparison Page</a>
```

Comment:
I'm unclear why there's both a calculate button and a link to the comparison page. I'd expect one or the other (probably the button), but not both.